build/ghost-buster/start.bash

```diff
@@ -10,10 +10,16 @@ THEMES="content/themes"
 
 cd "$GHOST"
 
-# Symlink data directory.
-mkdir -p "$OVERRIDE/$DATA"
+# Symlink data directory
+if [ ! -d "$OVERRIDE/$DATA" ]; then
+  mkdir -p "$OVERRIDE/$DATA"
+  for file in $DATA/*; do
+    cp -r "$file" "$OVERRIDE/$DATA"
```

Comment:
Ghost create database at install step, not at runtime. So we shouldn’t remove it after installation, just copy it to `/ghost-override`.